Cargo.toml

```
@@ -12,7 +12,7 @@ description = "A command-line utility to download songs and playlists from Spoti
 [dependencies]
 structopt = { version = "0.3", default-features = false }
 indicatif = "0.18"
-librespot = { version = "0.6.0", default-features = false }
+librespot = { version = "0.8.0" }
```

In order to keep librespot’s dependency set small, I suggest keeping default-features = false and explicitly selecting the TLS backend, as the build error suggests when it’s not set. If we don’t, librespot pulls in extra audio playback dependencies such as ALSA.

```suggestion
librespot = { version = "0.8.0", default-features = false, features = ["rustls-tls-native-roots"] }
```